src/session.rs

```diff
@@ -3031,8 +3041,19 @@ impl Session {
             Command::Edit(ref paths) => {
                 if paths.is_empty() {
                     self.unimplemented();
-                } else if let Err(e) = self.edit(paths) {
-                    self.message(format!("Error loading path(s): {}", e), MessageType::Error);
+                }
+
+                match self.edit(paths) {
+                    Ok((success_count, fail_count)) => self.message(
+                        format!(
+                            "Successfully loaded {} path(s), skipped {}",
+                            success_count, fail_count
+                        ),
+                        MessageType::Info,
```

Comment:
We should only show this message if `success_count + fail_count > 1`.

I'd also prefer something more concise, eg.: "4 path(s) loaded, 5 skipped"